tsconfig.json

```diff
@@ -1,20 +1,25 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "outDir": "lib",
+    "target": "es2015",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "strict": false,
     "forceConsistentCasingInFileNames": true,
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx",
+    "jsx": "react",
     "noImplicitAny": false
   },
-  "include": ["src"]
+  "exclude": [
+    "node_modules",
+    "**/*.stories.tsx",
+    "**/*.spec.ts"
```

Comment:
I think we should treat tests and tooling with the same respect as production code. Is it really bad in those files?

Reply:
Good point will make a issue for this